src/cli/commands/init/init.ts

```diff
@@ -130,6 +133,18 @@ export async function init(options: SWACLIConfig, showHints: boolean = true) {
 
     logger.log(`- Use ${chalk.cyan("swa deploy")} to deploy your app to Azure.\n`);
   }
+  const end = new Date().getTime();
+  const appFramework = projectConfig?.name?.split(", with")[0]!;
+  if (appFramework) {
+    await collectTelemetryEvent(TELEMETRY_EVENTS.Init, {
+      appFramework: appFramework,
```

Comment:
why do we need this if-else logic ? why cannot we have appFramework logged with undefined/null value as we are doing for api runtime property at other places?